docs/cli.rst

```diff
@@ -0,0 +1,988 @@
+^^^^^^^^^^^^^^^^^^^^
+Print the usage help for a command
+
+
+Usage
+^^^^^^^^^^^^^^^^^^^^
+
+::
+
+    $ nextflow help [options]
+
+
+Extended description
+^^^^^^^^^^^^^^^^^^^^
+This command is equivalent to simply issuing ``nextflow`` at the command line - it prints out the overview of the CLI interface and enumerates the top-level *options* and *commands*.
+
+
+Options
+^^^^^^^^^^^^^^^^^^^^
+
++---------------------------+------------+--------------------------------------------------------------------------------+
+| Name, shorthand (if any)  | Default    | Description                                                                    | 
++===========================+============+================================================================================+
+| -help, -h                 |  false     | Print the command usage.                                                       |
++---------------------------+------------+--------------------------------------------------------------------------------+
+
+
+
+Examples
+^^^^^^^^^^^^^^^^^^^^
+
+
+--------------------
+info          
+--------------------
+
+
+Description
+^^^^^^^^^^^^^^^^^^^^
+Print project and system runtime information
+
+
+
+Usage
+^^^^^^^^^^^^^^^^^^^^
+
+::
+
+    $ nextflow info [options]
+
+
+
+Extended description
+^^^^^^^^^^^^^^^^^^^^
+
+The ``info`` command prints out the nextflow runtime information about the hardware as well as the software versions of the ``Nextflow version and build``, ``Operating System`` and ``Groovy and Java runtime``.
+
+Options
+^^^^^^^^^^^^^^^^^^^^
+
++---------------------------+------------+--------------------------------------------------------------------------------+
+| Name, shorthand (if any)  | Default    | Description                                                                    | 
++===========================+============+================================================================================+
+| -check-updates, -u        |  false     | Check for remote updates.                                                      |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -d                        |  false     | Show detailed information.                                                     |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -help, -h                 |  false     | Print the command usage.                                                       |
++---------------------------+------------+--------------------------------------------------------------------------------+
+| -o                        |  text      | Output format, either ``text``, ``json`` or ``yaml``.                          |
++---------------------------+------------+--------------------------------------------------------------------------------+
+
+
+
+Examples
+^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
While trying out the options for the `info` command, I couldn't make use of the `-o` output feature.
```
(base) Abhinavs-MacBook-Pro:_resources eklavya$ nextflow info -o yaml
  Version: 20.07.1 build 5412
  Created: 24-07-2020 15:18 UTC (20:48 IDT)
  System: Mac OS X 10.15.6
  Runtime: Groovy 2.5.11 on OpenJDK 64-Bit Server VM 1.8.0_192-b01
  Encoding: UTF-8 (UTF-8)

(base) Abhinavs-MacBook-Pro:_resources eklavya$ nextflow info
  Version: 20.07.1 build 5412
  Created: 24-07-2020 15:18 UTC (20:48 IDT)
  System: Mac OS X 10.15.6
  Runtime: Groovy 2.5.11 on OpenJDK 64-Bit Server VM 1.8.0_192-b01
  Encoding: UTF-8 (UTF-8)
```

I've also tried coupling it with `-d` detailed option but the result always seems to be printed out in the `text` format. 

If this is the expected behavior, then perhaps I'm missing some info here 🤔